### PR TITLE
Fixes holodeck monkeys gibbing

### LINKED ISSUE #80369: Holodeck monkeys gib on despawn
Reporting client version: 514.1589

## Round ID:
[220557](https://scrubby.melonmesa.com/round/220557)

## Reproduction:

Spawn in the holodeck beach scene.
All the monkeys spawn in dead with zero organs. (this has already been reported in #79184 though maybe the cause has changed over time (?))

Despawn the holodeck scene.
All the monkeys gib (free limbs!) and ADMIN LOG: *no key*/(monkey (912) was gibbed after their chest teleport to Recreational Holodeck (144,158,4)

### PR DESCRIPTION
**Pull request by Time-Green**
Holodeck takes apart all hologram atoms, blindly. This breaks a lot, but most notably people.
I've also went and given monkeys organs again, which were removed in #79068. This broke incredibly hard, because monkeys can't live without DNA or organs (who would've thought)

Fixes #80369, #79184

:cl:
fix: Holodeck monkeys are properly cleaned up now
fix: Holodeck monkeys have organs now
/:cl:

Look they're so happy and wont gib on deload
image

### CODE REVIEW

File: code/modules/mob/living/carbon/human/monkey.dm [hunk 1]
```diff
@@ -45,6 +45,7 @@
 	. = ..()
 	for(var/obj/item/bodypart/limb as anything in bodyparts)
 		limb.bodypart_flags |= BODYPART_UNREMOVABLE // no farming organs or limbs from these fellers. get a monkey cube
+		limb.flags_1 = HOLOGRAM_1
```
Comment (MrMelbert) [inline code/modules/mob/living/carbon/human/monkey.dm:48]:
```suggestion
		limb.flags_1 |= HOLOGRAM_1
```
🗿

Comment (Time-Green) [reply code/modules/mob/living/carbon/human/monkey.dm]: Speedbrained

**Comment by Admiral-Gigaton** [comment]
Oh no my free organs!

**Comment by Time-Green** [comment]
Alright I've went ahead and fixed the holodeck monkeys completely. They now get organs and DNA. You can totally harvest the organs and limbs, and even transplant them, but they will still dissapear when leaving the holodeck.

You can technically give someone holodeck monkey organs to force them to stay on the holodeck or lose all their organs

(I know what everyone's thinking, but holograms are excluded from crafting and heretic rituals)

**Pull request merged**